src/umpire/config.hpp.in

```diff
@@ -7,7 +7,9 @@
 #ifndef UMPIRE_config_HPP
 #define UMPIRE_config_HPP
 
+#ifdef __cplusplus
 #include <string>
+#endif
```

Comment:
What is <string> used for in this file?

Reply:
Nothing technically, maybe just here: https://github.com/LLNL/Umpire/blob/ceb0f0ab0030861bda8b66f14937bba4180b8403/src/umpire/config.hpp.in#L40